docs/user/app_feature_intended.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Can we get something like this? 
![image](https://user-images.githubusercontent.com/9260483/183690944-a121fa2a-2c3d-4133-a320-4048e8146501.png)